### PR TITLE
Change the min SDK to 17

### PR DESCRIPTION
**Pull request by zihadrizkyef**

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+CurveGraphView
```
Comment (swapnil1104) [inline .idea/.name:1]: Please revert

File: .idea/gradle.xml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (swapnil1104) [inline .idea/gradle.xml:3]: Please revert these file changes

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8 (5)" project-jdk-type="JavaSDK">
```
Comment (swapnil1104) [inline .idea/misc.xml:3]: Please revert this too

**Comment by zihadrizkyef** [comment]
please check it again

**Pull request merged**